src/sentry/api/event_search.py

```diff
@@ -2318,6 +2319,12 @@ def result_type_fn(function_arguments, parameter_values):
                 ],
             ],
         ),
+        Function(
+            "divide",
+            required_args=[FunctionArg("numerator"), FunctionArg("denominator")],
+            aggregate=["divide", [ArgValue("numerator"), ArgValue("denominator")], None],
```

Comment:
This aggregate should look more like the following. However, due to a possible bug in snuba, the below query is being converted to `identity(NULL)`. Once that's resolved it should be okay.

```suggestion
            aggregate=[
                "divide",
                [
                    ArgValue("numerator"),
                    [
                        "if",
                        [
                            ["equals", [ArgValue("denominator"), 0]],
                            None,
                            ArgValue("denominator"),
                        ],
                    ],
                ],
                None
            ]
```